kvbc/src/Replica.cpp

```diff
@@ -65,7 +65,8 @@ Status Replica::initInternals() {
   m_currentRepStatus = RepStatus::Starting;
 
   if (replicaConfig_.isReadOnly) {
-    LOG_INFO(logger, "ReadOnly mode");
+    LOG_INFO(logger,
```

Comment:
nit: is there a reasib not to print it (with some change in style) outside for any replica in line 64?

Reply:
For backup replica's such information is printed just before this function call.